pos_partner_firstname/static/src/js/screens.js

```diff
@@ -0,0 +1,77 @@
+odoo.define('pos_partner_firstname.screens', function (require) {
```

Comment:
`pos_partner_firstname.screens` > `pos_partner_firstname.models`

Reply:
Ah! Right! I thought I did already ... anyway ... committed the fix :)